mail/backup_mailer/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		email-config-backup
+PLUGIN_VERSION=		1.0
+PLUGIN_COMMENT=		Send global configuration file backup by email
+PLUGIN_DEPENDS=		gnupg1 phpmailer
```

Comment:
I wanted to add these packages but have a small question: are you sure gnupg1 is a robust candidate for the foreseeable future? the current one seems to be https://www.freshports.org/security/gnupg

Reply:
Despite not supporting newer algorithms it has very few dependencies and for this specific use case I think gnupg1 is enough. But I still agree with you that its not the best "candidate for the foreseeable future". So should I change for gnupg?

Reply:
No, not at all, I just use 'gpg' command line program